packages/coinbase/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/coinbase",
-  "version": "2.1.3",
+  "version": "2.2.0",
```

```suggestion
  "version": "2.2.0-alpha.1",
```
This will allow for publishing of an alpha package for testing